docs/advanced/addons/vmimport.md

```diff
@@ -140,24 +140,36 @@ spec:
     namespace: default
     kind: VmwareSource
     apiVersion: migration.harvesterhci.io/v1beta1
+  gracefulShutdown: true
+  gracefulShutdownTimeout: 300000000000
 ```
 
-This will trigger the controller to export the VM named "alpine-export-test" on the VMWare source cluster to be exported, processed and recreated into the harvester cluster
+This will trigger the controller to export the VM named "alpine-export-test" on the VMware source cluster to be exported, processed and recreated into the harvester cluster
```

Comment:
```suggestion
This will trigger the controller to export the VM named "alpine-export-test" on the VMware source cluster to be exported, processed and recreated into the Harvester cluster
```

Reply:
Done incl. the versioned docs as well.